tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -2420,25 +2330,6 @@ public function adminRetrievesTheInformationOfUserUsingTheProvisioningApi(
 		);
 	}
 
-	/**
-	 * @Given /^the administrator has retrieved the information of user "([^"]*)"$/
```

Comment:
are these steps not used?

Reply:
No, these steps are not used in any feature files. There are more steps which are not in use. I have deleted some which I found while refactoring but there might be some other steps which aren't used.

Reply:
Do you feel like these steps can be of use in the near future? I'm not entirely sure if we should remove them, there might be occasions where these steps might come in handy but it also doesn't make sense to have some unused code lying around just so "there might be occasions where these steps might come in handy".

Why were these steps added in the first place do you have any idea?

We can remove them and re-implement them whenever they are needed I guess. Or any other suggestions from other members

cc @saw-jan @SagarGi @grgprarup @amrita-shrestha

Reply:
IMO we can follow YAGNI (You are not gonna need it) for now. If the step is required in future then the person can implement it again.